src/app/signing/signing.component.css

```
@@ -0,0 +1,13 @@
+
+#signing-box {
+	width: 50em;
+	height: 10em;
+	resize: none; /*both;*/
+	margin-top: 2em;
+	margin-left:auto;
+	margin-right:auto;
+}
+
+#options > * {
+	margin: 2em 0.5em;
+}
```

No newline at end of file
```suggestion
}

```